lib/vips/image.rb

```diff
@@ -692,6 +702,24 @@ def size
             [width, height]
         end
 
+        # Detect if image has an alpha channel
+        #
+        # @return [Boolean] true if image has an alpha channel.
+        def has_alpha?
+            return Vips::vips_image_hasalpha(self) != 0
+        end
+
+        # Append an alpha channel to an image.
+        #
+        # @return [Image] new memory image
+        def add_alpha
```

Comment:
Should this method be conditionally defined as well? Or just raise an `Vips::Error` if the vips version isn't high enough?

Reply:
I agree, let's have it as 8.6+ as well.

Reply:
Ok, I'll try to get the changes in today or tomorrow.

Reply:
Done!